src/graph/query_graph.c

```diff
@@ -88,7 +88,19 @@ static void _QueryGraphAddEdge
 			qg->unknown_reltype_ids = true;
 			continue;
 		}
-		array_append(edge->reltypeIDs, s->id);
+		// search if s-id exists in edge->reltypeIDs to don't insert duplicates ids
+		bool found = false;
```

Comment:
what happen if schema is not found we add the same name twice to `edge->reltypes`
then the plan is cached and after we create the data we will have the same problem

Reply:
You are right. I made some changes to prevent duplicates when schema is not found.